### ISSUE TITLE
host /dev/shm unmounted during build

### ISSUE THREAD

**Issue by ydirson** [ilbers/isar#76]
Using the same configuration as in https://github.com/ilbers/isar/issues/69#issuecomment-922343167:

```
user@isar-build:~/build$ bitbake mc:qemuamd64-buster:isar-image-base
...
NOTE: Executing Tasks
NOTE: Tasks Summary: Attempted 151 tasks of which 57 didn't need to be rerun and all succeeded.
user@isar-build:~/build$ bitbake mc:qemuamd64-buster:isar-image-base
FATAL: /dev/shm does not exist or is not writable
```

It's not the first time I'm hit by this issue, and every time there are so many targets run that it's hard to pinpoint the source of the problem.

`grep umount` in the logs brings something smelling bad, although it's hard to make a link with above issue, in `tmp/work/debian-buster-amd64/isar-bootstrap-target/1.0-r0/temp/log.do_bootstrap`:

```
Current default time zone: 'Etc/UTC'
Local time is now:      Thu Nov 11 13:32:38 UTC 2021.
Universal Time is now:  Thu Nov 11 13:32:38 UTC 2021.
Run 'dpkg-reconfigure tzdata' if you wish to change it.

umount: /home/user/build/tmp/work/debian-buster-amd64/isar-bootstrap-target/1.0-r0/rootfs/base-apt: no mount point specified.
DEBUG: Shell function do_bootstrap finished
```

That last one seems to come from `umount -l "${ROOTFSDIR}/base-apt" || true` in `isar-bootstrap.inc`, which should probably benefit from checking first before running `umount`, but I don't see that having a link with my problem.

I'd guess this comes from a script invoked by one of the isar recipes, but coming out from the isar repo.

**Comment by ismagulb** [comment]
Yes, we are aware of this. We provided a mount rework series to address it; unfortunately, we hit an issue with /proc/mounts entries sporadically disappearing on some reads and re-appearing on the next one. There is also a different rework implementation on the list; I don't know whether this issue is addressed there. What is your host? Is it bullseye?

As a workaround, I can suggest using kas-docker for now.

**Comment by ydirson** [comment]
I'm building on buster in this case.
